lib/motsu-proc/tests/default.rs

```diff
@@ -0,0 +1,94 @@
+mod erc20 {
+    use alloy_primitives::U256;
+    use stylus_sdk::stylus_proc::sol_storage;
+    sol_storage! {
+        #[derive(motsu_proc::DefaultStorageLayout)]
+        pub struct Erc20 {
+            mapping(address => uint256) _balances;
+            mapping(address => mapping(address => uint256)) _allowances;
+            uint256 _total_supply;
+        }
+    }
+
+    #[motsu::test]
+    fn erc20_initiates_correctly(contract: Erc20) {
+        assert_eq!(contract._total_supply.get(), U256::from(0));
+    }
+}
+
+mod uint {
+    use alloy_primitives::{U128, U16, U256, U32, U64, U8};
+    use stylus_sdk::stylus_proc::sol_storage;
+
+    sol_storage! {
+        #[derive(motsu_proc::DefaultStorageLayout)]
+        pub struct UintContract {
+            uint256 a;
+            uint128 b;
+            uint64 c;
+            uint32 d;
+            uint16 e;
+            uint8 f;
+            uint8 g;
+        }
+    }
+
+    #[motsu::test]
+    fn uint_initiates_correctly(contract: UintContract) {
+        assert_eq!(contract.a.get(), U256::from(0));
```

Comment:
If you want to use just `0`, use `U256::ZERO` and same for the rest of these types.

Reply:
Yeah, I was just initializing the other values with `USomething::from(value)` and forgot about zero lol. Thanks!